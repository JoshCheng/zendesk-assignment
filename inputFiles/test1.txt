3 4
Enter1 motorcycle SGX1234A2 1613541902
Enter car2 SGF9283P 1613541902
Exit SGX1234A3 1613545602
Enter car SGP2937F 1613546029
Exit SGP2937F 1613542901