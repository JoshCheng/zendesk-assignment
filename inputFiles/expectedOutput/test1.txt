Error occurred at line 2 - Invalid data.
Error occurred at line 3 - Invalid data.
Error occurred at line 4 - Invalid data.
Accept CarLot1
Error occurred at line 6 - Vehicle exit time is earlier or equal to enter time.
